app/grants/templates/grants/cart-vue.html

```diff
@@ -238,7 +241,7 @@ <h4 class="font-weight-bold mb-5 my-md-0">Discover Grants</h4>
         <b-dropdown-item-button>
           <div>
             <button class="btn p-0 checkout-logo" id='js-fundGrants-button'
-              @click="standardCheckout" :disabled="isCheckoutOngoing || grantsUnderMinimalContribution.length > 0"
+              @click="standardCheckout" :disabled="activeCheckout === 'standard' || grantsUnderMinimalContribution.length > 0"
             >
```

Comment:
Should the button not be disabled regardless which checkout process has been started?
Would this not make more sense?
```
:disabled="activeCheckout === undefined || grantsUnderMinimalContribution.length > 0"
```

Reply:
this would mean the buttons will be disabled by default (therefore unclickable) since `activeCheckout === undefined` is the default state

Reply:
Yes, sorry, I meant `activeCheckout === undefined` so that buttons are disable regardless which checkout process is running.